yangineer-simcity.md

```diff
@@ -0,0 +1,23 @@
+# Simcity
+The famous city from the game Simcity.
+Btw, the last Simcity sucked due to server issues, forced multiplayer mode, and limited city size.
```

Comment:
It's a city building simulator game made by the same creators of The Sims.
It used to be awesome, until the latest iteration where they forced multiplayer mode on every.